### ISSUE TITLE
small feature requests

### ISSUE THREAD

**Issue by Reload3r** [calzoneman/sync#523]
I don't think these should be too difficult to add and would help out quite a few channels.
1. Hide the video links to those below a certain rank, just display the name.
   This could shows what movie is next or even show people what videos are available without giving away all your movie links.
2. Allow 'Raw Video' or all videos to be renamed.
   I have too many 'raw videos' because i use some redirector links, they are impossible to manage right now.

**Comment by calzoneman** [comment]
1. I'll have to think about that one.  This kind of "security through obscurity" creates a false sense of security while it's still trivial for people to obtain the link of the currently playing video.  It creates extra work and I'm not sure it really accomplishes anything.
2. When you add a direct link video, another input box opens which allows you to set the title manually.  This is only shown for direct video links.  Editing the titles of existing videos is not currently supported, you will have to delete them from the playlist/library and re-add them.
   ![2015-10-29-210718_709x202_scrot](https://cloud.githubusercontent.com/assets/385672/10838279/132f7c64-7e81-11e5-97ab-77d697b19c4e.png)

**Comment by Reload3r** [comment]
You have a point that it could lead to false sense of security, but it would still require trolls that are mass-reporting/stealing videos to wait for each video to actually play instead of trolling everyones channel by mass reporting their playlist.

For the second point, i do not get this title box when i force urls with JW: , these links don't end with mp4 because they are behind a blogspot proxy or when using a google-redirector.
The videos work fine besides not having a title and I prefer doing this because it gives google drive videos the html5 player instead of the flash one :)

**Comment by calzoneman** [comment]
> For the second point, i do not get this title box when i force urls with JW: , these links don't end with mp4 because they are behind a blogspot proxy or when using a google-redirector.

Use of `jw:` is deprecated and will be removed as soon as I finish the mediaquery migration project.  If you want to override the extension check, use `fi:`.

**Comment by Reload3r** [comment]
Ah good to know!
Will fi: get title support though? Right now it doesn't for me.

**Comment by calzoneman** [comment]
Ah, it looks like there was a different mechanism being used to add the title input box.  I'll have to change that.

**Comment by Rexarood** [comment]
Hi calzoneman,
I've searched for a while now and this seems to be the best place to ask this because it continues this conversation.

My main problem is that I have videos that should be played chronologically but the naming makes it very unorganized. 

My first solution was to make separate playlists with a couple of videos each because I thought users could load up a playlist on my channel but I learned that the playlist feature is account-bound and not shareable. 

My next thought is to simply have all the videos looping on the channel but organized by name.
I really want to link a video URL and be able to change the name. I don't have the permissions to change the name of the file itself (on Google Drive) and putting in the URL doesn't give me the "rename" option like it does when you post a link with a file extension.

"If you want to override the extension check, use fi:."
"Ah, it looks like there was a different mechanism being used to add the title input box. I'll have to change that."
I'm not quite sure how to utilize this or if it even works since this update.

Any information would be helpful, thank you! Your service is amazing so far, keep it up!
